workflows/prognostic_c48_run/tests/test_tendency_prescriber.py

```diff
@@ -85,5 +84,8 @@ def add_one():
         derived_state["air_temperature"],
         (derived_state_copy["air_temperature"] + 2).assign_attrs(units="degK"),
     )
-    for variable in sorted(diags):
-        print(variable, joblib.hash(diags[variable]), file=regtest)
```

Comment:
I would leave it if possible since these provide a lot of coverage. Are you running these tests in a recent version of the prognostic run image? It is easy to get out of sync by using the `latest` tag.


Also using `.values` helps (see a7c278998462819d2ff36910d2cc19b6663827ed)

```
    for variable in sorted(diags):
        print(variable, joblib.hash(diags[variable].values), file=regtest)
```